pkg/service/rtcservice.go

```diff
@@ -276,6 +276,12 @@ func (s *RTCService) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		signalStats.ResolveRoom(join.GetRoom())
 		signalStats.ResolveParticipant(join.GetParticipant())
 	}
+	if pi.Reconnect && pi.ID != "" {
```

Comment:
isn't `ID` optional for reconnect? i vaguely remember some inconsistency here when we were rewriting migration

Reply:
Yeah, it was introduced later as a connect query param when we were implementing migrations. But, it has been around for close to 2 years. I think newer clients all send it.

Reply:
Looks like some clients may not be sending it. So, the signalling will not be resolved in those cases when it is running on a different node (both in OSS and cloud cases) and those bytes won't be counted. Something to chase down and fix on all clients.